### PR TITLE
Issue/#25 Add Ex Doc

### LINKED ISSUE #12: Add Documentation with [ExDoc](https://github.com/elixir-lang/ex_doc)

### PR DESCRIPTION
**Pull request by cloggy45**
Fixes #12 

Screen Shot 2019-06-02 at 21 10 12

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,4 @@ erl_crash.dump
 # Ignore package tarball (built via "mix hex.build").
 dms-*.tar
 
+.idea*
```
Comment (holsee) [inline .gitignore]: add a newline after statement

File: mix.exs [hunk 2]
```diff
@@ -23,7 +30,8 @@ defmodule DMS.MixProject do
   defp deps do
     [
       {:credo, "~> 1.0", only: [:dev]},
-      {:dialyxir, "~> 0.5.1", only: [:dev]}
+      {:dialyxir, "~> 0.5.1", only: [:dev]},
+      {:ex_doc, "~> 0.19", only: [:dev]},
```
Comment (holsee) [inline mix.exs]: add `runtime: false` for ex_doc dep

**Review by holsee — CHANGES_REQUESTED** [review]
Thanks @cloggy45, just a couple of minor tweaks needed.

**Comment by cloggy45** [comment]
PR Updated

**Review by holsee — APPROVED** [review]
LGTM

**Pull request merged**